index.html

```diff
@@ -108,59 +113,60 @@ <h2 class="p-title">
                 <li>2015</li>
             </ul>
-        </div>
 
-        <p class="about">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-        </p>
 
-        <ul class="languages">
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-        </ul>
-        <div><button class="clickme" type="button">See Projects</button></div>
-    </section>
+            <p class="about">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+            </p>
 
-    <section class="skills project-card">
+            <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <div><button class="clickme" type="button">See Projects</button></div>
+        </div>
+    </section>
+    <!--section4-->
+    <section class="project-card">
         <div class="image">
             <img src="./images/portfolio3.png" alt="Image3" />
-            <div class="stories">
-                <h2 class="p-title">Tonic</h2>
-                <ul class="subwork">
-                    <li class="work1">CANOPY</li>
-                    <li>
-                        <img src="./images/Counter.svg" alt="" />
-                    </li>
-                    <li>Back End Dev</li>
-                    <li>
-                        <img src="./images/Counter.svg" alt="" />
-                    </li>
-                    <li>2015</li>
-                </ul>
+        </div>
+        <div class="background">
+            <h2 class="p-title">Tonic</h2>
+            <ul class="subwork">
+                <li class="work1">CANOPY</li>
+                <li>
+                    <img src="./images/Counter.svg" alt="" />
+                </li>
+                <li>Back End Dev</li>
+                <li>
+                    <img src="./images/Counter.svg" alt="" />
+                </li>
+                <li>2015</li>
+            </ul>
 
-                <p class="about">
-                    A daily selection of privately personalized reads; no accounts or
-                    sign-ups required.
-                </p>
+            <p class="about">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+            </p>
 
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>JavaScript</li>
-                </ul>
-                <div><button class="clickme" type="button">See Projects</button></div>
-            </div>
+            <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <div><button class="clickme" type="button">See Projects</button></div>
         </div>
     </section>
```

Comment:
- Kindly add some top and button spacing between each project to match the Figma design in the desktop version.
- Kindly make the left and right spacing for each project identical in the mobile version to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/155775046-3abbe8be-00c5-4abf-8ae3-c4cfad96ec31.png) | ![image](https://user-images.githubusercontent.com/24703171/155775123-f2036374-3e4c-4799-9780-2605b8e2b4d3.png) |